cmake/developer_package/frontends/frontends.cmake

```diff
@@ -121,7 +140,8 @@ macro(ov_add_frontend)
     # Create named folders for the sources within the .vcproj
     # Empty name lists them directly under the .vcproj
 
-    source_group("src" FILES ${LIBRARY_SRC})
+    ov_frontend_group_files(${frontend_root_dir}/ "src" "cpp")
+    ov_frontend_group_files(${frontend_root_dir}/ "src" "proto")
```

Comment:
maybe we need a generic function which accepts ROOT_FOLDER and:
- understands all its sub-folders
- replicates `source_group` based on those subfolders